src/components/EnergyConsumptionChart/index.tsx

```diff
@@ -156,33 +157,34 @@ const EnergyConsumptionChart = () => {
     ],
   })
 
-  const aspectRatioValue = useBreakpointValue({
-    base: 0.55,
-    sm: 0.75,
-    md: 1.1,
-  })
+  // reverse data if RTL
+  if (isRtl) {
+    rawData = rawData?.reverse()
+  }
 
   // chart options config
   const chartOptions = {
     // chart styles
-    barThickness: 38,
-    borderRadius: 4,
     responsive: true,
-    aspectRatio: aspectRatioValue,
-    maintainAspectRatio: true,
```

Comment:
Removed the `aspectRatio` and leave it to adapt automatically to the container size.